### ISSUE TITLE
[FEAT] Support `/watch?v=` URLs

### ISSUE THREAD

**Issue by gBasil** [snuffyDev/Beatbump#179]
**Is your feature request related to a problem? Please describe.**
When you want to go from music.youtube.com or youtube.com to beatbump.io, you not only have to replace the domain, but also `watch?v=` with `listen?id=`.

**Describe the solution you'd like**
It would be nice if Beatbump had support for the path YouTube uses. This could take the form of:
1. Redirecting to `listen?id=` from `watch?v=` for parity.
2. Redirecting to `watch?v=` from `listen?id=`.

**Additional context**
LibRedirect—an extension for redirecting some sites to other frontends such as Beatbump—seems to take care of the URL transformation, but it would be nice if it was supported in Beatbump itself.

**Comment by snuffyDev** [comment]
Good idea! I’ll see what I can do. Was thinking of having a watch page equivalent for Beatbump, and keeping ‘/listen’ as a share page
